### PR TITLE
Add mosaic page

### PR DESCRIPTION
**Pull request by harin**
Add mosaic page with basic examples from https://github.com/jtr13/codehelp/blob/master/R/mosaic.md and added `ggplot`, `vcd::doubledecker`, and some theory.

Todo
- [ ] Add banner
- [ ] Generate .md file?

### CODE REVIEW

**Comment by harin** [comment]
@zachbogart I would like to get some feedback on the mosaic page and how I should proceed.

1. Do I need to generate the .md or do you have a deployment pipeline that does that automatically?
2. Where do I get the banner for the page?
3. Is there anything you feel that needs to improve or is missing? 
4. Is there an easy way to preview the bookdown in development easily? (I tried to build the book in rstudio but it keeps saying missing library, then when I installed it, another will show up one at a time)

**Comment by zachbogart** [comment]
- Please sync your fork with the main repo: https://help.github.com/articles/syncing-a-fork/
- You have an include chunk option early in your code. False must be in all caps.

**Comment by harin** [comment]
@harin

**Comment by harin** [comment]
@zachbogart I have fixed the issues

**Comment by jtr13** [comment]
It's getting stuck now on ggmosaic. The library needs to be added to the DESCRIPTION file.

File: DESCRIPTION [hunk 1]
```diff
@@ -25,5 +25,5 @@ Imports:
   viridis,
   viridisLite,
   visNetwork,
+  ggmosaic
```
Comment (jtr13) [inline DESCRIPTION]:
```suggestion
  ggmosaic,
```

**Comment by jtr13** [comment]
You're missing the comma after ggmosaic.  I made a suggestion (a new GitHub feature) to add the comma, which you can approve. If it's not clear how to do that, let me know.

File: DESCRIPTION (hunk 1 above)
Comment (harin) [reply DESCRIPTION]: Thank you

**Comment by zachbogart** [comment]
I did a pass over your chapter and it looks great. Added a banner and edited some wording, but looks good to merge.

**Pull request merged**